lib/Expo.php

```diff
@@ -73,6 +78,13 @@ public function unsubscribe($interest, $token = null)
     {
         return $this->registrar->removeInterest($interest, $token);
     }
+    
+    /**
+     * @param string|null $accessToken
+     */
+    public function setAccessToken(?string $accessToken = null) {
```

Comment:
Could you remove `?` from the method signature cause I believe that doesn't work with php7.0 which is still supported in the package 🤷‍♂️

Reply:
Yeah, you're right :)